server/server.go

```diff
@@ -56,12 +53,17 @@ func Listen(cfg *config.Configuration, handler http.Handler, adminHandler http.H
 		go runServer(mainServer, "Main", mainListener)
 	}
 
-	var adminListener net.Listener
-	if adminListener, err = newTCPListener(adminServer.Addr, nil); err != nil {
-		glog.Errorf("Error listening for TCP connections on %s: %v for admin server", adminServer.Addr, err)
-		return
+	if cfg.Admin.Enabled {
```

Comment:
Is it possible to add a simple test in `server_test.go` to cover this new configurable `admin`?

Reply:
Discussed offline with @SyntaxNode, this comment isn't a concern.